app/helpers/notification_count.rb

```diff
@@ -2,12 +2,17 @@ module ExercismWeb
   module Helpers
     module NotificationCount
       def notification_count
```

Comment:
I kind of like the idea of a tool that enforces a certain kind of code style, maybe we could tweak the rubocop config file?